UKHO.PeriodicOutputService/UKHO.BESS.BuilderService/UKHO.BESS.BuilderService.csproj

```diff
@@ -6,10 +6,10 @@
     <Nullable>enable</Nullable>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
-    <NoWarn>1701;1702;8600;8602;8618</NoWarn>
+    <NoWarn>1701;1702;8600;8602;8618;8604</NoWarn>
```

Comment:
What error is being suppressed here?

Reply:
8604 warning is suppressed for null reference argument  for parsing the correlationId